admin-dev/themes/new-theme/scss/components/_grid.scss

```diff
@@ -322,15 +359,27 @@ table {
           max-width: 6rem;
         }
       }
-    }
-  }
-}
 
-// -----------------------
-// Styling grid extensions
-// -----------------------
+      td[data-column-id^="id_product"],
```

Comment:
Hardcoded values like this do not look good. Can we do anything on the Grid level to help you style it with a more generic class?

Reply:
I removed those IDs; it should work the same without any breaking changes ✅
However, the IDs were already used on certain elements in this file, but you're right, it's not ideal.